### DISCUSSION TITLE
Connection between mobile devices

### DISCUSSION THREAD

**Discussion by jrubiano01** [innoveit/react-native-ble-manager#1173] [Q&A]
I'm using the example app and I do a quick scan, I want to try to connect to a mobile device, I have the connection code of the other device, but when I look for it in the list, it doesn't appear, is the connection between mobile devices possible or not?

**Comment by marcosinigaglia** [comment]
The simple answer is no. This library offers the 'client' side of BLE so you cannot use it to connect two mobile devices.

**Reply by jrubiano01** [reply]
I have another question, I'm trying to connect to two SmartWatches, one has features to take the pulse and the other doesn't, I've noticed that I've only been able to establish connection with the first option while the other doesn't accept it, does that mean that only some BLE devices are "Compatible"?

**Reply by jrubiano01** [reply]
And if so, I'm using the example app and I'd like you to tell me what the method is to get the data or what would be the exact method or event for that?
